packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -177,36 +238,54 @@ const FooterTagList = ({post, classes, hideScore, hideAddTag, smallText=false, s
           </MaybeLink>
         : null
       )
-    ) 
+    )
 
-  if (loadingInitial || !results) {
-    return <span className={classes.root}>
-     {sortTags(post.tags, t=>t).map(tag => <FooterTag key={tag._id} tag={tag} hideScore smallText={smallText}/>)}
-     {!hidePostTypeTag && postType}
-    </span>;
-  }
+  const sortedTagRels = results ? sortTags(results, t=>t.tag).filter(tagRel => !!tagRel?.tag) : []
 
-  const sortedTagRels = sortTags(results, t=>t.tag).filter(tagRel => !!tagRel?.tag)
+  const innerContent =
+    (loadingInitial || !results) ? (
```

Comment:
For this part it might be useful to look at it side by side. This used to be two separate returns but now they are both wrapped in the same outer span